### DISCUSSION TITLE
Feature request: support multiple user accounts

### DISCUSSION THREAD

**Discussion by SamBouwer** [victorbalssa/abacus#233] [Ideas]
It would be awesome if it is possible to login with a second firefly account and easily switch between them. Configuration between user accounts could be partially shared (for example, firefly url) and partially separate (for example, auth/PAT).

**Comment by victorbalssa** [comment]
Hey @SamBouwer 

I created a new issue #235 to surface this in our roadmap, I will try to scope and include it in a milestone

**Comment by victorbalssa** [comment]
Done -> closing
